src/components/NavBar.js

```diff
@@ -77,3 +82,4 @@ const mapStateToProps = (state) => {
   };
 };
 export default withRouter(connect(mapStateToProps, actions)(NavBar));
+//austin.mehmet@villagebookbuilders.org
```

Comment:
This should definitely not be here!

Reply:
Yeah , I had it there before you added the persistence. I must of forgot to remove it after , due to not having to use it. Thanks for that Paul.